src/cogent3/core/alignment.py

```diff
@@ -2813,7 +2814,9 @@ def to_html(
         name_order
             order of names for display.
         interleave_len
```

Comment:
fore the docstring just use `will be replaced by wrap in version 2021.6` and delete the rest